### ISSUE TITLE
The MenuItem role doesn't support the aria-expanded state

### ISSUE THREAD

**Issue by aristov** [w3c/aria-practices#629]
The [WAI-ARIA Roles, States, and Properties](https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties-12) section of the Menu or Menubar design pattern contains the following statement:

> A parent menuitem has aria-expanded set to false when its child menu is not visible and set to true when the child menu is visible.

But according to WAI-ARIA-1.1 [the MenuItem role doesn't support this state](https://www.w3.org/TR/wai-aria/#menuitem).
